sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -215,6 +227,16 @@ public static Read readResources() {
     return new Read();
   }
 
+  /**
+   * Search resources from a PCollection
+   *
+   * @return the search
```

Comment:
nit: you can remove these if they are obvious. Same for other comments.

Reply:
Keeping this one for consistency with other methods, removed some other comments.